package.json

```diff
@@ -87,7 +88,7 @@
   },
   "homepage": "https://github.com/shakacode/react_on_rails#readme",
   "dependencies": {
-    "react-on-rails": "^9.0.0-beta.12",
+    "react-on-rails": "^9.0.3",
```

Comment:
Why does React on Rails version 10 require React on Rails version 9?

Reply:
@Judahmeek This should be fixed.

Reply:
@shepmaster Until we release, we can't update the version in the PR.

Reply:
> Until we release 

I noticed it because the 10.0 release of this package requires the 9.0 release of this package 😸

Reply:
It seems [react-on-rails@10.0.0](https://www.npmjs.com/package/react-on-rails) is released.
I send a PR to update this version. https://github.com/shakacode/react_on_rails/pull/959